src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -123,7 +208,65 @@ def build_nodes(self) -> List[NodeDescription]:
 
     def build_well(self) -> WellDescription:
         """Create the description for the well."""
+        well_length = self.get_position_in_well(self.score_input.read_general_data()["final_md"])
+        operation_data = self.score_input.read_operation_data()
+        initial_bottom_pressure = operation_data["flow_initial_pressure"].GetValue(PRESSURE_UNIT)
+        # the factor multiplied for the top pressure is arbitrary, just to give a initial value
+        initial_top_pressure = 0.6 * initial_bottom_pressure
         return attr.evolve(
             super().build_well(),
             pvt_model=self.get_fluid_model_name(),
+            initial_conditions=InitialConditionsDescription(
+                pressures=InitialPressuresDescription(
+                    position_input_type=TableInputType.length,
+                    table_length=PressureContainerDescription(
+                        positions=Array([0.0, well_length.GetValue()], LENGTH_UNIT),
+                        pressures=Array(
+                            [initial_top_pressure, initial_bottom_pressure], PRESSURE_UNIT
+                        ),
+                    ),
+                ),
+                volume_fractions=InitialVolumeFractionsDescription(
+                    position_input_type=TableInputType.length,
+                    table_length=VolumeFractionsContainerDescription(
+                        positions=Array([0.0], LENGTH_UNIT),
+                        fractions={
+                            FLUID_GAS: Array([0.1], FRACTION_UNIT),
+                            FLUID_OIL: Array([0.9], FRACTION_UNIT),
+                            FLUID_WATER: Array([0.0], FRACTION_UNIT),
+                        },
+                    ),
+                ),
+                velocities=InitialVelocitiesDescription(
+                    position_input_type=TableInputType.length,
+                    table_length=VelocitiesContainerDescription(
+                        positions=Array([0.0], LENGTH_UNIT),
+                        velocities={
+                            FLUID_GAS: Array([0.0], VELOCITY_UNIT),
+                            FLUID_OIL: Array([0.0], VELOCITY_UNIT),
+                            FLUID_WATER: Array([0.0], VELOCITY_UNIT),
+                        },
+                    ),
+                ),
+                temperatures=InitialTemperaturesDescription(
+                    position_input_type=TableInputType.length,
+                    table_length=TemperaturesContainerDescription(
+                        positions=Array([well_length.GetValue()], LENGTH_UNIT),
```

Comment:
Here you could use the formation temperature.
Keep, the bottom position/temperature as it was included, however include the top position (0 m) and the top temperature as defined in environment.

Reply:
Ok, sounds good, done.

Reply:
With this change the case converges ok.